FirelandsProject/firelands-cata
# Feat (core\conf): LFG Solo

### NEED INDEPENDENT TESTING

### Changes proposed

- Implement my project's group original module SOLO LFG as a Core Feature
- Allows player to enter instantly solo or in a group that does not meet team requirements for LFG AND RDF
- Conf in world server to have on or off
- This was original designed as a Debug support module to allow developers who did not have a human group to enter instances thru lfg to test their work without having to constant being in gm mode.

### Issues addressed


- Closes

### How to Test the Changes:


1.  ensure conf is added and enabled LFGSolo.Enabled = 1
2. ensure you have a level appropiate player and select a instance to enter thru lfg
3. select your roles
4. observer you entered the instances with meeting team requirements

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
